cci_nepal/analysis/dummy_data/dummy_data_modeling/model_analysis.py

```diff
@@ -0,0 +1,295 @@
+import pandas as pd
+import numpy as np
+from numpy import absolute
+from numpy import mean
+from numpy import std
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import accuracy_score
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import cross_val_score
+from sklearn.model_selection import RepeatedKFold
+from sklearn.feature_selection import SequentialFeatureSelector
+from matplotlib import pyplot
+from operator import itemgetter
+
+import logging
+import cci_nepal
+from cci_nepal.getters.dummy_data import get_dummy_data as gdd
+
+
+def label_transformer(d):
```

Comment:
Can you add doc-strings to each function and some comments to explain where need (you may be able to describe it enough just using the doc string as they are quite short).